docs/source/workflow_integration/databricks_visualisation.md

```diff
@@ -1,12 +1,22 @@
-# How to run Kedro-Viz on Databricks
+# Visualise a Kedro project in Databricks notebooks
 
-[Kedro-Viz](../visualisation/kedro-viz_visualisation.md) is a tool that allows you to visualise your Kedro pipeline. It is a standalone web application that runs on a web browser, it can be run on a local machine or on Databricks itself.
+[Kedro-Viz](../visualisation/kedro-viz_visualisation.md) is a tool that allows you to visualise your Kedro pipeline. It is a standalone web application that runs on a web browser, it can be run on a local machine or in Databricks notebooks.
 
 For Kedro-Viz to run with your Kedro project, you need to ensure that both the packages are installed in the same scope (notebook-scoped vs. cluster library). This means that if you `%pip install kedro` from inside your notebook then you should also `%pip install kedro-viz` from inside your notebook.
 If your cluster comes with Kedro installed on it as a library already then you should also add Kedro-Viz as a [cluster library](https://docs.microsoft.com/en-us/azure/databricks/libraries/cluster-libraries).
 
+To run Kedro-Viz on Databricks you must first [launch the Kedro IPython extension](./databricks_workspace.md#9-using-the-kedro-ipython-extension).
+
 Kedro-Viz can then be launched in a new browser tab with the `%run_viz` line magic:
 
 ```ipython
 In [2]: %run_viz
```

Comment:
Should we remove the `In [2]:` here?

Reply:
I think so, either that or show it as a screen capture so it's clear that it comes from the notebook and is not something you type in.